modules/components/src/Aggs/BooleanAgg.js

```diff
@@ -54,6 +54,11 @@ export default ({
     field: dotField,
   });
 
+  const isTrueBucketDisabled =
+    trueBucket === undefined || trueBucket?.doc_count <= 0;
+  const isFalseBucketDisabled =
+    falseBucket === undefined || falseBucket?.doc_count <= 0;
```

Comment:
Correct me if I'm wrong :P

I think we just need the `trueBucket?.doc_count <= 0` and `falseBucket?.doc_count <= 0` case, since if `trueBucket === undefined`, then the latter will always be `true`